installation/cmd/run.sh

```diff
@@ -195,12 +196,36 @@ if [[ ${DUMP_DB} ]]; then
         sed -i 's/image\:.*compass-schema-migrator.*/image\: k3d-kyma-registry\:5001\/compass-schema-migrator\:'$DUMP_IMAGE_TAG'/' ${ROOT_PATH}/chart/compass/templates/update-expected-schema-version-job.yaml
     fi
 
+    REMOTE_VERSIONS=($(gsutil ls -R gs://sap-cp-cmp-dev-db-dump/ | grep -o -E '[0-9]+' | sed -e 's/^0\+//' | sort -r))
+    LOCAL_VERSIONS=($(ls migrations/director | grep -o -E '^[0-9]+' | sed -e 's/^0\+//' | sort -ru))
```

Comment:
By mistake this var was not deleted/edited and the job keeps failing with the same error because this var is processed first. As the correct syntax is in the var below, this one can be deleted.